metadata.yaml

```diff
@@ -27,6 +33,14 @@ requires:
   prometheus-target:
     interface: http
   prometheus-rules:
-    interface: prometheus-rules
+    interface: prometheus_rules
   dashboards:
-    interface: grafana-dashboard
+    interface: grafana_dashboard
+  nrpe-external-master:
+    interface: nrpe_external_master
+  general-info:
+    interface: general_info
+  monitors:
+    interface: monitors
+  local-monitors:
+    interface: local_monitors
```

Comment:
The interface in the NRPE charm uses dashes https://git.launchpad.net/charm-nrpe/tree/metadata.yaml#n27